### ISSUE TITLE
Unable to create new step with error 'Failed to find templates dir/box: could not locate box "templates"'

### ISSUE THREAD

**Issue by abotkin-cpi** [bitrise-io/bitrise#667]
Installed Bitrise CLI v1.29.0 via Homebrew and attempted to run `bitrise :step create`.  After filling in the information about my step, GitHub repo, etc, I eventually end up with the following error:

```
Creating Step directory at: /Users/abotkin/src/bitrise-step-xcode-set-simulator-language
Error: Failed to write template into file: Failed to evaluate template: Failed to find templates dir/box: could not locate box "templates"
Usage:
  bitrise-plugin-step create [flags]

Flags:
  -h, --help   help for create

ERROR: Failed to write template into file: Failed to evaluate template: Failed to find templates dir/box: could not locate box "templates"
FATA[21:21:44] Failed to run plugin (step), error: exit status 255 
```

EDIT: Tried on another machine & saw the identical error after upgrading Homebrew's Bitrise CLI from 1.28.0 to 1.29.0.  The same machine had been able to create a step with v1.28.0 just minutes before Bitrise was updated.

**Comment by bitce** [comment]
Hi @abotkin-cpi,

Sorry for the inconvenience, but thanks a lot for the detailed report! :) Forwarded to our Tooling team, they'll check it out and we'll get back to this thread ASAP!

**Comment by mackoj** [comment]
I have the same issue 😄

**Comment by mackoj** [comment]
Do you have an eta on this on ?

**Comment by fehersanyi-bitrise** [comment]
Hello @mackoj if everything goes well, the tooling team can pick up the task next week.

**Comment by mackoj** [comment]
@bitce thanks

**Comment by abotkin-cpi** [comment]
This appears to be resolved with Bitrise v1.30.0
